airflow/providers/amazon/aws/secrets/secrets_manager.py

```diff
@@ -264,14 +286,14 @@ def get_conn_uri(self, conn_id: str) -> str | None:
 
     def get_variable(self, key: str) -> str | None:
         """
-        Get Airflow Variable from Environment Variable
+        Get Airflow Variable
```

Comment:
Can you add some more description here. If we're getting this variable from multiple places maybe we should be clear about the options?

Reply:
In the context of this class, we are actually getting it from only one location: AWS Secrets Manager. If this function returns None, then it is fetched from other location: Environment variable then metastore but this is done outside of this class. [See documentation here](https://airflow.apache.org/docs/apache-airflow/1.10.10/howto/use-alternative-secrets-backend.html). Let me know if you still think I should update the documentation